### PR TITLE
Jv lottery/solution

### PR DESCRIPTION
**Pull request by VitaliiLeshchenko**

### CODE REVIEW

File: src/main/java/core/basesyntax/Ball.java [hunk 1]
```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+public class Ball {
+    private static int number;
+    private String color;
+
+    public Ball(String color) {
+        this.color = color;
+        Ball.number++;
```
Comment (overpathz) [inline src/main/java/core/basesyntax/Ball.java:9]: Ball should accept the color and number (black ball with 8 number).
In this case you just counting your balls

**Review by overpathz — CHANGES_REQUESTED** [review]

File: src/main/java/core/basesyntax/Colors.java [hunk 2]
```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+public enum Colors {
```
[the remark this answers is not in the document]
Comment (VitaliiLeshchenko) [reply src/main/java/core/basesyntax/Colors.java:3]: Colors in upper case, like "COLORS" ?

File: src/main/java/core/basesyntax/Lottery.java [hunk 3]
```diff
@@ -0,0 +1,7 @@
+package core.basesyntax;
+
+public class Lottery {
+    public Ball getRandomBall() {
```
[the remark this answers is not in the document]
Comment (VitaliiLeshchenko) [reply src/main/java/core/basesyntax/Lottery.java:10]: done

File: src/main/java/core/basesyntax/Ball.java (hunk 1 above)
Comment (VitaliiLeshchenko) [reply src/main/java/core/basesyntax/Ball.java]: done

File: src/main/java/core/basesyntax/Ball.java [hunk 4]
```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
```
[the remark this answers is not in the document]
Comment (VitaliiLeshchenko) [reply src/main/java/core/basesyntax/Ball.java:1]: done

File: src/main/java/core/basesyntax/Colors.java (hunk 2 above)
[the remark this answers is not in the document]
Comment (VitaliiLeshchenko) [reply src/main/java/core/basesyntax/Colors.java:3]: i get it
enum values must be in upper case

[the remark this answers is not in the document]
Comment (VitaliiLeshchenko) [reply src/main/java/core/basesyntax/Colors.java:3]: done

**Pull request closed without merging**